### PR TITLE
Create a django template

### PR DESCRIPTION
**Pull request by dleitee**
Create the first django template solving the checkboxes on the issue #2

### CODE REVIEW

File: project_name/project_name/urls.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Including another URLconf
+    1. Import the include() function: from django.conf.urls import url, include
+    2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
+from django.conf.urls import url
+from django.contrib import admin
+
+urlpatterns = [
+    url(r'^admin/', admin.site.urls),
+    url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework'))
```
Comment (smaniotto) [inline project_name/project_name/urls.py]: Can we have a comment on this URL, explaining that it's used to login on the browsable API? 😬

**Comment by smaniotto** [comment]
👏

File: project_name/project_name/urls.py (hunk 1 above)
Comment (jonatasbaldin) [reply project_name/project_name/urls.py]: Sure!

**Comment by jonatasbaldin** [comment]
Hey everyone I just assigned to this PR, could you guys give your feedback? Is it good to merge?

We need this basic template to start working on the other features!

With love ❤️

**Review by joaoricardo000 — APPROVED** [review]
👏

**Comment by jonatasbaldin** [comment]
Some claps are enough for us!

:shipit:

**Pull request merged**